s3tests_boto3/functional/test_s3select.py

```diff
@@ -282,18 +333,94 @@ def test_nullif_expressions():
 
     nose.tools.assert_equal( res_s3select_nullif, res_s3select)
 
+    res_s3select_nullif = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name,"select (nullif(_1,_2) is null) from stdin ;")  ).replace("\n","")
+
+    res_s3select = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name,"select (_1 == _2) from stdin  ;")  ).replace("\n","")
+
+    nose.tools.assert_equal( res_s3select_nullif, res_s3select)
```

Comment:
as general note.
/ nose.tools.assert_equal( res_s3select_nullif, res_s3select) /
in case both queries are correct but return empty result (and it should return something, or something is wrong with input or query)
the assert is succeed, no alert on that.

the nose.tools.assert_equal  should replace by functionality which validate that results are not empty and equal.